### PR TITLE
John2

### PR DESCRIPTION
**Pull request by johnwroge**
Changed two routes in server.ts for clarity (get and post for /user are now post methods with /login and /register). Implemented bcrypt for storing passwords in db. Added fetch for login component. 

Co-authored-by: Harvey Li <112590580+harvli@users.noreply.github.com>
Co-authored-by: John Wroge <72668920+johnwroge@users.noreply.github.com>
Co-authored-by: Aimée Nguyen <106037255+DoctorCodeine@users.noreply.github.com>
Co-authored-by: Ed Cho <106274379+edcho720@users.noreply.github.com>

### CODE REVIEW

File: dist/src/server/database/db.js [hunk 1]
```diff
@@ -20,41 +20,12 @@ mongoose
...
     email: { type: String, required: true },
-    username: { type: String, required: true },
     password: { type: String, required: true },
 });
 //Collection name is Users
-const User = mongoose.model('Users', userSchema);
+const Users = mongoose.model('Users', userSchema);
 // Export db
-module.exports = User;
+module.exports = Users;
```
Comment (harvli) [inline dist/src/server/database/db.js:31]: Great job on the Bcrypt John! Did you mean to include the db.js file?

**Review by harvli — CHANGES_REQUESTED** [review]
Awesome job on the bcrypt implementation and cleaning up your code! Doubling checking if we need the db.js

File: dist/src/server/database/db.js (hunk 1 above)
Comment (johnwroge) [reply dist/src/server/database/db.js:31]: Thanks! I didn't realize I sent it again. It was deleted from my branch a while ago. Is it possible to remove it?

Comment (johnwroge) [reply dist/src/server/database/db.js:31]: Just confirming there's no db.js file in my repo or branch. So I don't think merging will effect anything.

**Review by johnwroge — COMMENTED** [review]
Deleted.

**Review by harvli — APPROVED** [review]
Nice job!

**Pull request merged**